File: LibraryManagement.API/DTOs/Response/MemberResponse.cs
namespace LibraryManagement.API.DTOs.Response;

public class MemberResponse
{
    public int Id { get; set; }

    public string FirstName { get; set; } = string.Empty;

    public string LastName { get; set; } = string.Empty;

    public string FullName { get; set; } = string.Empty;

    public string DocumentNumber { get; set; } = string.Empty;

    public string Email { get; set; } = string.Empty;

    public string PhoneNumber { get; set; } = string.Empty;

    public bool IsActive { get; set; }

    public int LoanCount { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }
}

File: LibraryManagement.Frontend/src/app/core/models/response/member-response.ts
export interface MemberResponse {
  id: number;
  firstName: string;
  lastName: string;
  fullName: string;
  documentNumber: string;
  email: string;
  phoneNumber: string;
  isActive: boolean;
  loanCount: number;
  createdAt: string;
  updatedAt?: string;
}